src/components/ChartIndex.tsx

```diff
@@ -1,13 +1,15 @@
 import React from "react";
```

Comment:
I removed this file as part of the latest cleanup.
Since this is not adding much value
Please import Sessions directly in homePage.tsx and invoke it there.

Reply:
done